forge-gui/res/cardsfolder/a/absorb_energy.txt

```diff
@@ -2,7 +2,7 @@ Name:Absorb Energy
 ManaCost:1 U U
 Types:Instant
 A:SP$ Counter | TargetType$ Spell | TgtPrompt$ Select target spell | ValidTgts$ Card | RememberForCounter$ True | SubAbility$ DBAnimate | SpellDescription$ Counter target spell.
-SVar:DBAnimate:DB$ AnimateAll | Zone$ Hand | Duration$ Perpetual | ValidCards$ Card.YouOwn+sharesCardTypeWith Remembered | staticAbilities$ ReduceCost | SubAbility$ DBCleanup | SpellDescription$ Cards in your hand that share a card type with that spell perpetually gain "This spell costs {1} less to cast."
+SVar:DBAnimate:DB$ AnimateAll | Zone$ Hand | Duration$ Perpetual | ValidCards$ Card.YouOwn+sharesCardTypeWith Remembered | StaticAbilities$ ReduceCost | SubAbility$ DBCleanup | SpellDescription$ Cards in your hand that share a card type with that spell perpetually gain "This spell costs {1} less to cast."
```

Comment:
uuh you can't just change these?
for some reason the effects have it coded as starting with lower caps

Reply:
Oops, my bad. I should've asked about it before being so bold.

The confusion that lead into this mishap stemmed from there being a very similar parameter, `StaticAbilities$`, using a good number of the same arguments as `staticAbilities$`. Now looking more closely, the latter is nearly exclusively associated with  `Animate` effects (85 hits), while the other is much more frequent (904 hits) if also closely associated with (presumably) generic `Effect` effects. I do hope my changing `sVars$` instances  to  `SVars$` isn't as problematic.
In any case, unless there is interest in disambiguating these more clearly, I'll backtrack whatever is deemed unnecessary.

Reply:
Yeah  the smaller case annoyed me too

You just need to update them in Animate Effect too

Reply:
yea might be worth just making the change in Java too

Reply:
I take it these edits will stand but at Hanmac's request they've been moved here, https://github.com/Card-Forge/forge/pull/4753